src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline

```diff
@@ -1637,7 +1637,7 @@ public class LibraryClient implements BackgroundResource {
    * Sample code:
    * <pre><code>
    * try (LibraryClient libraryClient = LibraryClient.create()) {
-   *   ResourceName name = ShelfName.of("[SHELF_ID]");
+   *   ResourceName name = ArchivedBookName.of("[ARCHIVE_PATH]", "[BOOK_ID]");
```

Comment:
Here too, `ListStringsResponse.name` is listed as an AnyResourceNameConfig in the library_gapic.yaml, so this change is semantically alright.